src/app.js

```diff
@@ -1,21 +1,30 @@
 import { div } from '@cycle/dom'
+import isolate from '@cycle/isolate'
 import xs from 'xstream'
+import { Instrument } from './components'
 
-export function App() {
-  const vtree$ = xs.of(
-    div('My Awesome Cycle.js app'),
-  )
+export function App({ DOM$ }) {
+  const randomFrequency = () => Math.round(Math.random() * 1000) + 200
```

Comment:
This function should not be into App function because it doesn't need anything from App